src/validation/json-schemas/jwk/general-jwk.json

```diff
@@ -0,0 +1,96 @@
+{
+  "$id": "https://identity.foundation/dwn/json-schemas/general-jwk.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "required": ["kty"],
+  "properties": {
+    "alg": {
+      "type": "string"
+    },
+    "kid": {
+      "type": "string"
+    },
+    "kty": {
+      "enum": ["EC", "RSA", "oct", "OKP"]
+    },
+    "crv": {
+      "type": "string"
+    },
+    "use": {
+      "type": "string"
+    },
+    "key_ops": {
+      "type": "string"
+    },
+    "x5u": {
+      "type": "string"
+    },
+    "x5c": {
+      "type": "string"
+    },
+    "x5t": {
+      "type": "string"
+    },
+    "x5t#S256": {
+      "type": "string"
+    }
+  },
+  "oneOf": [
+    {
+      "properties": {
+        "kty": {
+          "const": "EC"
```

Comment:
For my understanding: why is it that `d` is not included in the "generic" JWK schema?